api_tests/tests/e2e/bitcoin_ethereum.ts

```diff
@@ -0,0 +1,497 @@
+/**
+ * @config { "ledgers": ["bitcoin", "ethereum"], "logDir" : "bitcoin_ethereum" }
+ */
+
+import { twoActorTest } from "../../lib/actor_test_new";
+import { AssetKind } from "../../lib/asset";
+import { sleep } from "../../lib/utils";
+import { expect } from "chai";
+
+// ******************************************** //
+// Bitcoin/bitcoin Alpha Ledger/ Alpha Asset    //
+// Ethereum/ether Beta Ledger/Beta Asset        //
+// ******************************************** //
+describe("Bitcoin/bitcoin - Ethereum/ether", () => {
```

Comment:
The tests are pretty verbose now (again) with the move of `it` to the outside of the `twoActorTest` function. I like that we can now run them through the IDE but it would also be nice if you could try and consolidate some of those function calls again.

The actual test is 4 indentation levels in :(

Reply:
Mhm... just thinking out lout: 
What if we start 2 cnd (or depending how many we need) per test suite/test file in a `beforeAll` block? 

This might speed up our tests and stress test cnd 😬
(not sure if this is a good idea as long as we have flaky tests though... meaning forever)